spring-cloud-netflix-core/src/test/java/org/springframework/cloud/netflix/ribbon/apache/RibbonLoadBalancingHttpClientTests.java

```diff
@@ -451,7 +451,7 @@ public void testDoubleEncodingWithRetry() throws Exception {
 			public boolean matches(Object argument) {
 				if(argument instanceof HttpUriRequest) {
 					HttpUriRequest arg = (HttpUriRequest)argument;
-					return arg.getURI().equals(uri);
+					return arg.getURI().getPath().equals(uri.getPath());
```

Comment:
I am concerned for this modification and am interested if you have any suggestions on why it is necessary.  Note that it is caused by the ribbon client updating the request with a new URI

Reply:
Actually I think I messed up the URI in this test, see https://github.com/spring-cloud/spring-cloud-netflix/pull/2669

Reply:
Thanks!  I will look forward to updating after #2669 is merged